code/datums/uplink/uplink_item.dm

```diff
@@ -71,8 +77,8 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	var/cost = 0
 	var/last = 0 // Appear last
 	var/abstract = 0
-	var/list/gamemodes = list() // Empty list means it is in all the gamemodes. Otherwise place the gamemode name here.
-	var/list/excludefrom = list() //Empty list does nothing. Place the name of gamemode you don't want this item to be available in here. This is so you dont have to list EVERY mode to exclude something.
+	var/list/on_uplink_type = list() //Empty means it is in all uplink types. Otherwise place the uplink type defined in code\datums\uplink\uplinks.dm here.
```

Comment:
```suggestion
	var/list/on_uplink_type = list() //Empty means it is in all uplink types. Otherwise place the uplink type defined in code/__DEFINES/misc.dm here.
```

I moved this but didn't update docs